### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#49549]
#### VFS team name
Benefits Team 1

#### VFS product name
Claim Status Tool

#### VFS feature name
Download Decision Letters

#### Point of Contact/Reviewers
Tiffany Pender - @TiffanyPender - Accessibility (Shared Support team)

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
Heading levels should only increase by one. The page has a single H1 (correct), but then jumps to an H4 (claim letter heading).

### Link, screenshot or steps to recreate
These should be `h2`s, not `h4`s 😄 
<img width="345" alt="Screen Shot 2022-11-10 at 1 45 53 PM" src="https://user-images.githubusercontent.com/14154792/201180367-91c3066d-d411-4480-9bd4-e4d57dc3d96d.png">

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
- [x] @jaredcunha Update the claim letter headings ("Letter dated...") to be H2s since they follow an H1.
- [x] @jaredcunha Use utility classes (if needed) to maintain the visual size of the headings

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jaredcunha** [comment]
This is resolved. Here is a screen shot from staging.
<img width="1062" alt="Screen Shot 2022-11-28 at 4 06 52 PM" src="https://user-images.githubusercontent.com/1094951/204381032-9f0ba5c1-82fe-4c56-ad6c-30ba3c4577e0.png">

**Comment by shiragoodman** [comment]
FYI @jaredcunha please be sure to close out the ticket when the issue is resolved. This is a VFS responsibility (see VFS Guidance section). We are still showing this as an open issue.

**Comment by jstrothman** [comment]
Not sure when this was fixed, but headings are now properly nested
